cmd/saturn-l2/main.go

```diff
@@ -114,6 +114,8 @@ var (
 	saturn_l1_hostName = "saturn-test.network"
 
 	defaultL1DiscoveryURL = "https://orchestrator.saturn-test.network/nodes/nearby"
+
+	waitL1Connectivity = 10 * time.Second
```

Comment:
Let's make this interval shorter please, I am proposing 3 seconds.

```suggestion
	waitL1Connectivity = 3 * time.Second
```